build-tools/create-dotnet-pkg/create-dotnet-pkg.csproj

```diff
@@ -0,0 +1,86 @@
+<Project Sdk="Microsoft.Build.NoTargets">
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <AppendTargetFrameworkToOutputPath>false</AppendTargetFrameworkToOutputPath>
+    <OutputPath>..\..\bin\Build$(Configuration)</OutputPath>
+  </PropertyGroup>
+
+  <Import Project="..\..\Configuration.props" />
+  <Import Project="..\scripts\XAVersionInfo.targets" />
+
+  <PropertyGroup>
+    <PkgInstallDir>/</PkgInstallDir>
+    <PayloadDir>$(OutputPath)\pkg\archive</PayloadDir>
+    <PkgOutputPath>$(OutputPath)\pkg\packages</PkgOutputPath>
+    <PkgResourcesPath>$(OutputPath)\pkg\resources</PkgResourcesPath>
+    <PkgDistributionDestination>$(OutputPath)\pkg\distribution.xml</PkgDistributionDestination>
+    <LicenseDestination>$(PkgResourcesPath)\en.lproj</LicenseDestination>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <None Remove="**" />
+    <None Include="distribution.xml.in" />
+  </ItemGroup>
+
+  <UsingTask AssemblyFile="$(PrepTasksAssembly)" TaskName="Xamarin.Android.BuildTools.PrepTasks.ReplaceFileContents" />
+
+  <Target Name="_FinalizePayload"
+      DependsOnTargets="GetXAVersionInfo">
+    <ItemGroup>
+      <_Licenses Include="$(DotNetPreviewPath)sdk-manifests\$(DotNetPreviewVersionBand)\Microsoft.NET.Workload.Android\License*" />
+    </ItemGroup>
+    <PropertyGroup>
+      <License Condition=" '$(License)' == '' and '@(_Licenses->Count())' == '1' ">@(_Licenses)</License>
```

Comment:
Wait, so… if `@(_License)` ever contains *more than one* file, we instead default to the top-level MIT License? That seems odd?

Reply:
I will change this to do this instead, which should grab the last one:
```xml
<License Condition=" '$(License)' == '' and '@(_Licenses->Count())' != '0' ">%(_Licenses.Identity)</License>
```

Reply:
I'm not sure that's necessarily an improvement?

If there should *never* be more than one License file, should we `<Error/>` on it?

If there *should* be more than one…but we only ever use one…wtf should happen?

This just "feels weird".

Reply:
The file in the `.nupkg` will either be `License.txt` from monodroid or `LICENSE` from xamarin-android.

I could rename the file to always be `LICENSE`, I suppose.